index.html

```diff
@@ -12,32 +12,207 @@
+                    <ul class="list-container">
+                        <li class="project-list">html</li>
+                        <li class="project-list">bootstrap</li>
+                        <li class="project-list">ruby</li>
+                    </ul>
+
+                </div>
+                <button class="button card-button rmv" type="button">See Project</button>
+            </li>
+            <li class="project-card banker">
+                <div class="text-container">
+                    <h3 class="project-title">Profesional Art Printing Data</h3>
+                    <p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="list-container">
+                        <li class="project-list">html</li>
+                        <li class="project-list">bootstrap</li>
+                        <li class="project-list">ruby</li>
+                    </ul>
+                </div>
+                <button class="button card-button rmv" type="button">See Project</button>
+            </li>
+            <li class="project-card bank">
+                <div class="text-container ">
+                    <h3 class="project-title">Profesional Art Printing Data</h3>
+                    <p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="list-container">
+                        <li class="project-list">html</li>
+                        <li class="project-list">bootstrap</li>
+                        <li class="project-list">ruby</li>
+                    </ul>
                 </div>
+                <button class="button card-button rmv" type="button">See Project</button>
+            </li>
+        </ul>
+    </section>
+    <section class="about-section">
+        <h2 class="about-title">About me</h2>
+        <div class="about">
+            <p class="about-text">Hello I’m a software developer! I can help you build a product ,<br> feature or website Look through some of my work and experience! <br>If you like what you see and have a project you need coded, <br>don’t hestiate to contact me.</p>
+            <button class="button about-button" type="button">Get My Resume</button>
+        </div>
+        <div><img src="images/Vector4.png" alt="line logo" class="line"></div>
+            <ul class="language">
+            <li><img src="images/Rectangle55.png" alt="my logo" class="img">Language
+                <ul>
+                    <li class="language-list">Javascript</li>
+                    <li class="language-list">Ruby</li>
+                    <li class="language-list">Html</li>
+                    <li class="language-list">css</li>
+                </ul>
+            </li>
```

Comment:
Plese kindly fix the padding, width and margin issues in your about section because it looks different from the template. And also use the correct font sizes for this please again pay attention to detail and carefully inspect the template to get it right.

Your design:
![Screenshot from 2022-05-31 18-25-07](https://user-images.githubusercontent.com/79506105/171202709-296c9910-02c7-4873-b136-a4dd728f3363.png)

Follow this figma design: 
![Screenshot from 2022-05-31 18-29-31](https://user-images.githubusercontent.com/79506105/171202811-03006adc-5c3d-43bc-bb90-4d3e1e8e644a.png)

Kindly position your back-ground image properly
Also you the correct font family and font-size

Reply:
done as requested by the reviewer